apps/condo/domains/organization/constants/errors.js

```diff
@@ -1,14 +1,16 @@
 const ALREADY_ACCEPTED_INVITATION = 'ALREADY_ACCEPTED_INVITATION'
 const ALREADY_INVITED = 'ALREADY_INVITED'
-const EMPTY_NAME_ERROR = '[name.is.too.short'
+const NAME_TOO_SHORT_ERROR = '[name.is.too.short'
+const NAME_VALUE_INVALID = '[name.is.too.invalid'
```

Comment:
where did you see the notation? 😄

Reply:
<img width="443" alt="image" src="https://user-images.githubusercontent.com/46228212/192264890-ea42428d-c36b-44a1-a348-3d7ca73bbad6.png">
Errors with TIN are written in a similar notation. I take it this is something old?

Reply:
I think this it rudiment. Some time ago we used to write errors something like this: 
`[model:field:shortInfo] Detailed message`

Right now we\re trying to translate front-end errors. That's why it's seems strange for me to meet this notation here. `WHATS_GOING_ON` notation is more preferable for me. But I\m a bit outdated here. Maybe it's better to ask in dev chat about this.

Reply:
fixed